### PR TITLE
Inline scan script

### PR DESCRIPTION
**Pull request by yathi-sysdig**
Script to analyze container images built locally.

### CODE REVIEW

File: inline_scan.sh [hunk 1]
```diff
@@ -0,0 +1,562 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
```
Comment (markjacksonfishing) [inline inline_scan.sh]: Please set this as ` set -euo pipefail`

File: inline_scan.sh [hunk 2]
```diff
@@ -0,0 +1,562 @@
...
+    if [[ "${#IMAGE_FILES[@]}" -ge 1 ]] || [[ -f /tmp/sysdig/sysdig_output.log ]]; then
+        if [[ -d "/tmp/sysdig" ]]; then
+            rm -rf "/tmp/sysdig"
+        fi
+    fi
+
+    exit "${ret}"
+}
+
+main "$@"
```
Comment (markjacksonfishing) [inline inline_scan.sh:562]: Please add a line feed here

**Review by markjacksonfishing — CHANGES_REQUESTED** [review]

File: inline_scan.sh (hunk 1 above)
Comment (yathi-sysdig) [reply inline_scan.sh]: done

File: inline_scan.sh (hunk 2 above)
Comment (yathi-sysdig) [reply inline_scan.sh:562]: done

File: inline_scan.sh (hunk 1 above)
Comment (markjacksonfishing) [reply inline_scan.sh]: `eou`

File: inline_scan.sh (hunk 2 above)
Comment (markjacksonfishing) [reply inline_scan.sh:562]: I do not see a line feed after line 562

**Review by markjacksonfishing — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,59 @@
...
+     
+      Script for performing analysis on local docker images, utilizing the Sysdig analyzer subsystem.
+      After image is analyzed, the resulting Anchore image archive is sent to a remote Sysdig installation
+      using the -r <URL> option. This allows inline_analysis data to be persisted & utilized for reporting.
+     
+      Images should be built & tagged locally.
+     
+        Usage: inline_scan.sh analyze -s <SYSDIG_REMOTE_URL> -k <API Token> [ OPTIONS ] <FULL_IMAGE_TAG>
+     
+          -s <TEXT>  [required] URL to Sysdig Scanning API endpoint (ex: -r 'https://secure-sysdig.com')
```
Comment (marojor) [inline README.md]: minor typo in the example (should be `-s` instead of `-r`).

**Review by drosen0 — COMMENTED** [review]
The script works well. I suggest exiting with `2` (or anything other than `1`) for errors, so that `1` can be reserved for the scanner found a vulnerability (when `scan` mode and `-f` specified).

**Review by markjacksonfishing — APPROVED** [review]

**Pull request merged**